backend/svs/query_presets.py

```
@@ -739,6 +739,20 @@ class _QuickPresetList:
+        label="CNV dominant",
+        inheritance=Inheritance.AFFECTED_CARRIERS,
+        frequency=Frequency.RELAXED,
+        impact=Impact.EXONIC,
+        sv_type=SvType.CNVS_EXTRA_LARGE,
+        chromosomes=Chromosomes.WHOLE_GENOME,
+        regulatory=Regulatory.DEFAULT,
+        tad=Tad.DEFAULT,
+        known_patho=KnownPatho.DEFAULT,
+        genotype_criteria=GenotypeCriteriaDefinitions.SVISH_HIGH,
+        database=Database.REFSEQ,
+    )
```

Ruff (0.15.12)

[warning] Do not perform function call `QuickPresets` in dataclass defaults

(RUF009)